sixtyfps_compiler/passes/compile_paths.rs

```diff
@@ -98,8 +98,16 @@ pub fn compile_paths(
                             element_name
                         ),
                     };
-                    let bindings = std::mem::take(&mut child.borrow_mut().bindings);
-                    path_data.push(PathElement { element_type, bindings });
+
+                    if child.borrow().repeated.is_some() {
+                        diag.push_error(
+                            "Path elements are not supported with `for`-`in` syntax".into(),
```

Comment:
should we say "yet" and link to the issue?
That way people can be invovled

Reply:
Sure, I was a bit hesitant but if we're certain that we're going to support this eventually then it makes sense.